src/api/job.js

```diff
@@ -0,0 +1,56 @@
+const {Router} = require('express');
+const Job = require('../persistence/job');
+
+const router = new Router();
+
+// Admin Create a new job with post method
```

Comment:
**The rule is to try to NOT comment in your code.**
- If your code is clear, don't need comment
- If your code is complicated that you think you have to explain it, refactor the complicated code instead.

Reply:
I fixed it.